### PR TITLE
wimlib 1.8.0, minor formula improvements

### PR DESCRIPTION
**Pull request by timsutton**
- remove require 'formula', update to sha256
- add a test to create a wimfile and get info on it

### CODE REVIEW

**Comment by timsutton** [comment]
I didn't notice the difference with the `osxfuse` dep since I happened to have already it installed on the Yosemite machine I was working on. Is there a better way I can handle this case in the Formula?

Thanks!

**Comment by DomT4** [comment]
Apologies this didn't get a reply till now. I recently pushed a version update alongside a change of handling to the `fuse` dependency due to us moving `fuse`-requiring formulae out of the Core. If you want to rebase this on master and then improve the test, please feel free to do so!

**Comment by timsutton** [comment]
Thanks! I'm glad the fuse dependency issue got sorted out. My last commit just adds the test I'd added in the previous commit.

File: Library/Formula/wimlib.rb [hunk 1]
```diff
@@ -31,6 +31,16 @@ def install
   end
 
   test do
-    system bin/"wiminfo", "--help"
+    # make a directory containing a dummy 1M file
+    Dir.mkdir("foo")
```
Comment (DomT4) [inline Library/Formula/wimlib.rb]: You don't need the `Dir` here. Just `mkdir "foo"` will work fine :).

**Comment by DomT4** [comment]
One nit but otherwise looks good :+1:

**Comment by timsutton** [comment]
Thanks! Showing my rusty Ruby :)

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/e75b01c98bbb45be8ad8ebe32956f2eaa2237683. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**